parameters_processing/02_parameter_upload.Rmd

```diff
@@ -169,153 +169,117 @@ These are the final columns:
-  filter(experiment == "2nd_field_biomass", 
-         treatment == "5cm")
-
-sla_5cm <- left_join(leaf_area_5cm, leaf_biomass_5cm, by = "plant_id") %>% 
-  mutate(sla_initial_units = leaf_area_cm2 / leaf_dry_biomass_g, 
-         dry_biomass_kg = ud.convert(leaf_dry_biomass_g, "g", "kg"), 
-         area_m2 = ud.convert(leaf_area_cm2, "cm2", "m2"), 
-         SLA = area_m2 / dry_biomass_kg) %>% 
-  mutate(local_datetime = as.Date(`harvest_time`), 
-         treatment = "outdoor 5 cm density") %>% 
-  select(local_datetime, treatment, SLA) %>% 
-  mutate(statname = '')
+sla_5cm <- greenhouse_outdoor_biomass %>%
+  filter(genotype == "ME034V-1",
+        exp_site == "Field", 
+        exp_number ==  2, 
+        !is.na(leaf_DW_g), 
+        !is.na(leaf_area_cm2),
+        treatment == "5cm") %>%
+  mutate(sla_initial_units = leaf_area_cm2 / leaf_DW_g, 
+         leaf_DW_kg = ud.convert(leaf_DW_g, "g", "kg"), 
+         leaf_area_m2 = ud.convert(leaf_area_cm2, "cm2", "m2"), 
+         SLA = leaf_area_m2 / leaf_DW_kg,
+         Treatment = "outdoor 5 cm density",
+         local_datetime = harvest_date) %>% 
+  select(local_datetime, Treatment, SLA) 
 
 # Biomass and leaf area for 2nd field experiment, JollyG
-leaf_biomass_JollyG <- read_excel(data_path2, sheets_names2[7]) %>% 
-  dplyr::rename(leaf_dry_biomass_g = 16) %>% # leaf DW was entered as g instead of mg
-  filter(treatment == "jolly_pot") %>% 
-  mutate(leaf_dry_biomass_g == as.numeric(leaf_dry_biomass_g))
-
-leaf_area_JollyG <- read_excel(data_path2, sheets_names2[1]) %>% 
-  dplyr::rename(leaf_area_cm2 = 9) %>% 
-  filter(experiment == "2nd_field_biomass", 
-         treatment == "jolly_p")
-
-sla_JollyG <- left_join(leaf_area_JollyG, leaf_biomass_JollyG, by = "plant_id") %>% 
-  mutate(sla_initial_units = leaf_area_cm2 / leaf_dry_biomass_g, 
-         dry_biomass_kg = ud.convert(leaf_dry_biomass_g, "g", "kg"), 
-         area_m2 = ud.convert(leaf_area_cm2, "cm2", "m2"), 
-         SLA = area_m2 / dry_biomass_kg) %>% 
-  mutate(local_datetime = as.Date(`harvest_time`), 
-         treatment = "outdoor JollyG soil") %>% 
-  select(local_datetime, treatment, SLA) %>% 
-  mutate(statname = '')
+sla_JollyG <- greenhouse_outdoor_biomass %>%
+  filter(genotype == "ME034V-1",
+        exp_site == "Field", 
+        exp_number ==  2, 
+        !is.na(leaf_DW_g), 
+        !is.na(leaf_area_cm2),
+        treatment == "jolly_pot") %>%
+  mutate(sla_initial_units = leaf_area_cm2 / leaf_DW_g, 
+         leaf_DW_kg = ud.convert(leaf_DW_g, "g", "kg"), 
+         leaf_area_m2 = ud.convert(leaf_area_cm2, "cm2", "m2"), 
+         SLA = leaf_area_m2 / leaf_DW_kg,
+         Treatment = "outdoor JollyG soil",
+         local_datetime = harvest_date) %>% 
+  select(local_datetime, Treatment, SLA) 
 ```
 
-Combineand format all SLA values for upload. 
+Combine and format all SLA values for upload. 
 ```{r}
 sla <- bind_rows(sla_hightemp, sla_control, sla_gh, sla_5cm, sla_JollyG) %>%
```

Comment:
Comparing the old version to this, they were identical except for a one day shift for the outdoor data. That seems like a minor difference, just wanted to document it. 
![Screen Shot 2021-01-29 at 9 14 40 AM](https://user-images.githubusercontent.com/6686508/106299590-92fe3a00-6212-11eb-9d1e-fc8227e687f4.png)

Reply:
Noted. Good eye!